python/lib/sift_py/ingestion/config/telemetry.py

```diff
@@ -220,17 +221,26 @@ def _from_yaml(
                         }
                     )
 
+            if rule.get("asset_names") or rule.get("tag_names"):
+                logger.warning(
```

Comment:
Having now thought about this a bit more could you mind just adding a `TODO` note to emit a warning for now? I haven't had a chance yet to think about the logging/deprecation story for this lib yet. I personally hate it when I use library that emit a bunch of logs I have no control over because it adds to noise. The perfect solution will be to have a logger instance (and it's cihldren) scoped to this library that users can configure in terms of severity level and where the logs go (stdout, stderr, file, or `/dev/null`, etc.). This is something we can do in the future. Could you remove the dependency in the mean time as well?

Reply:
Sure, seems alright to me for now. I feel like it'd be good for us to continue those conversations pretty soon on this just to make sure we're giving clear and ample feedback to users about what's going on sooner than later